models/interactives.go

```diff
@@ -25,7 +25,7 @@ type Interactive struct {
 	SHA          string `bson:"sha,omitempty"       json:"sha,omitempty"`
 	FileName     string `bson:"file_name,omitempty" json:"file_name,omitempty"`
 	State        string `bson:"state,omitempty"     json:"state,omitempty"`
-	Active       bool   `bson:"active,omitempty"    json:"active,omitempty"`
+	Active       *bool  `bson:"active,omitempty"    json:"active,omitempty"`
```

Comment:
Do we need the pointer? I'm thinking we can get rid of `omitempty` for a normal bool to work ok. 

I get why the pointer - because you can have `nil` and `false` is not the same as `nil`. But we will always set this value - so true or false is fine.

Also the string attribs - these will never be empty too so do we need `omitempty` here?

Reply:
limitation of the golang mogo-db api, update of a boolean only happens if a pointer is passed to it (otherwise it sees as "no change" and rejects the update request.